### ISSUE TITLE
[BUG] Encryption key requires decryption key

### ISSUE THREAD

**Issue by jcweber10** [Mastercard/client-encryption-java#76]
#### Bug Report Checklist

- [X] Have you provided a code sample to reproduce the issue?
- [X] Have you tested with the latest release to confirm the issue still exists?
- [X] Have you searched for related issues/PRs?
- [X] What's the actual output vs expected output?

**Description**
I'm running into an issue in the `JWEConfigBuilder` where either the `decryptionKey` or `encryptionCertificate` is required. I want to use an `encryptionKey` instead of `encryptionCertificate`, and I don't want to be forced to provide a `decryptionKey`. Please correct me if I'm wrong, but I assume this was just a minor oversight in #60.

**To Reproduce**
1. Use the `JWEConfigBuilder` to create a `JWEConfig` with an `encryptionKey` and no `decryptionKey`.
2. `IllegalArgumentException("You must include at least an encryption certificate or a decryption key")` is thrown

**Expected behavior**
The `JWEConfig` can be built without any exceptions being thrown, and the config can be used to perform encryption.

**Sample failing code**
```
val jweConfig = JweConfigBuilder
            .aJweEncryptionConfig()
            .withEncryptionKey(publicKey)
            .withEncryptionPath("$", "$")
            .withEncryptedValueFieldName("encryptedPayload")
            .build()
```

**Additional context**
I'm using this library in an Android app that makes calls to middleware, which makes the calls to Mastercard APIs. I want to avoid worrying about managing the private key in my app, which is why I'd like to see this resolved. In the meantime I plan on using an empty implementation of the `PrivateKey` interface, but I'd like to avoid that if possible. 

**Related issues/PRs**
Has a similar issue/PR been reported/opened before? No, but it seems related to #60 

**Suggest a fix/enhancement**
Changing the implementation of `checkParameterValues` in `JweConfigBuilder` should fix this issue.
```
private void checkParameterValues() {
        if (decryptionKey == null && encryptionCertificate == null && encryptionKey) {
            throw new IllegalArgumentException("You must include at least an encryption certificate, an encryption key, or a decryption key");
        }
    }
```

**Comment by rfeelin** [comment]
Hi @jcweber10 ,
Thanks for bringing this to our attention. We'll put it in our backlog for now and let you know once we've released a fix.

**Comment by rfeelin** [comment]
Hi @jcweber10 ,
Thanks for bringing this to our attention. We'll put it in our backlog for now and let you know once we've released a fix.

**Comment by danny-gallagher** [comment]
@jcweber10, Thanks again for reporting this bug, it will be fixed in [release 1.7.11](https://github.com/Mastercard/client-encryption-java/releases/tag/v.1.7.11)
